src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/TdsParserStateObject.netcore.cs

```diff
@@ -842,145 +842,6 @@ internal void WriteByte(byte b)
             _outBuff[_outBytesUsed++] = b;
         }
 
-        internal Task WriteByteSpan(ReadOnlySpan<byte> span, bool canAccumulate = true, TaskCompletionSource<object> completion = null)
```

Comment:
Note: this is removed in the netcore/netfx specific files, and the netcore implementation is now in the shared TdsParserStateObject class